MapleServer2/PacketHandlers/Game/SkillHandler.cs

```diff
@@ -125,7 +125,7 @@ private static void HandleSyncSkills(GameSession session, PacketReader packet)
             packet.ReadInt();
             packet.ReadByte();
 
-            session.FieldManager.BroadcastPacket(SkillSyncPacket.Sync(skillSN, session.FieldPlayer, position, rotation, toggle));
+            session.Send(SkillSyncPacket.Sync(skillSN, session.FieldPlayer, position, rotation, toggle));
```

Comment:
wait no I meant it should be:
`session.FieldManager.BroadcastPacket(SkillSyncPacket.Sync(skillSN, session.FieldPlayer, position, rotation, toggle), session);`
So every player would get it except the caster

Reply:
oh ok